requirements.txt

```diff
@@ -12,7 +12,7 @@ coloredlogs
 query_phenomizer
 Flask-Babel>=3
 livereload
```

Comment:
```suggestion
livereload>=2.7
```

Reply:
Yay, livereload have updated during summer! We should add a version requirement on it as well, as otherwise we can end up at an impasse with a newer tornado and an old livereload which still has the old WSGIContainer issue (https://github.com/lepture/python-livereload/pull/275). Like this:
![Screenshot 2024-08-19 at 10 11 41](https://github.com/user-attachments/assets/8f21eacf-f7db-4145-9140-fa2910041418)

Reply:
Ah right! I must have had a newer version then! Thanks!